src/core/src/validation_util.cpp

```diff
@@ -1669,3 +1669,10 @@ bool ov::is_rank_compatible_any_of(const ov::Rank& rank, const std::vector<Rank>
         return rank.compatible(r);
     });
 }
+
+bool ov::validate_tensor_vector(const TensorVector& tensor_vector, const size_t& size) {
+    return (tensor_vector.size() == size) &&
+           std::none_of(tensor_vector.cbegin(), tensor_vector.cend(), [](const Tensor& tensor) {
```

Comment:
Since the `TensorVector` keeps the tensors by value, is it even possible to get a `nullptr` here? IMO this check can be removed.

Reply:
Thanks, check removed